modules/installing-rhv-setting-up-ca-certificate.adoc

```diff
@@ -14,25 +14,33 @@ Later, you provide the certificate to the installation program.
 
 .Procedure
 
-. Download the CA certificate using either of these methods:
-** Go to the Manager's webpage and click the *CA Certificate* link. The URL of the Manager's webpage uses this format: `https://<engine-fqdn>/ovirt-engine/`.
+. Use either of these two methods to download the CA certificate:
+** Go to the Manager's webpage, `https://<engine-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
 ** Run the following command:
 +
 ----
 $ curl -k 'https://<engine-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  <1>
 ----
 <1> For `<engine-fqdn>`, specify the fully qualified domain name of the {rh-virtualization} Manager, such as `rhv-env.virtlab.example.com`.
 
-. Add the certificate to the computer using the appropriate method for your operating system:
-** For Linux, copy the CA certificate to the directory for server certificates, and update the CA trust. For example:
+. For Linux, copy the CA certificate to the directory for server certificates
 +
 ----
 $ sudo cp /tmp/ca.pem /etc/pki/ca-trust/source/anchors/ca.pem
-$ update-ca-trust
 ----
+. Configure the CA file to grant rootless user access to the Manager. Set the CA file permissions to have an octal value of `0644` (symbolic value: `-rw-r--r--`). For example, run:
```

Comment:
Is this a Linux-only step? If so, I might combine this step and the previous one into substeps under something like "If you use Linux, modify the CA file permissions:"

Reply:
This step applies to both Mac and Linux. I've reordered the steps and added the `-p` option to one of them to make the flow clearer.